doc/00_Linee guida interoperabilità tecnica/02_ambito-di-applicazione.rst

```diff
@@ -4,22 +4,20 @@ Ambito di applicazione
 Le Linee Guida sull’interoperabilità tecnica delle Pubbliche 
 Amministrazioni sono redatte in ottemperanza:
 
--  alla lettera a comma 2 dell’articolo 14-bis del DECRETO LEGISLATIVO 
-   7 marzo 2005, n. 82 (Codice dell'amministrazione digitale), che 
-   individua le funzioni di AgID tra cui: "emanazione di Linee [...] 
+-  alla lettera a comma 2 dell’articolo 14-bis del Decreto Legislativo 
+   7 marzo 2005, n. 82 (Codice dell'Amministrazione Digitale, CAD), che 
+   individua le funzioni dell'Agenzia per l'Italia Digitale (AgID) tra cui: "emanazione di Linee [...] 
    di indirizzo [...] sull'attuazione e sul rispetto delle norme di cui 
    al presente Codice, anche attraverso l'adozione di atti amministrativi 
    generali, in materia di [...] interoperabilità e cooperazione 
    applicativa tra sistemi informatici pubblici e quelli dell'Unione 
    europea";
 
-- alla lettera b comma 3-ter dell'articolo 73 del DECRETO LEGISLATIVO 
-   7 marzo 2005, n. 82 (Codice dell'amministrazione digitale), in cui 
-   è indicaco che il Sistema pubblico di connettività è costituito da 
+- alla lettera b comma 3-ter dell'articolo 73 del CAD, in cui 
```

Comment:
@vintra73 se abbiamo già definito CAD direi di usare l'acronimo, che per il lettore è cmq più chiaro.

Reply:
la tabella dei termini è in un capitolo successivo... cmq per me non ci sono problemi se decidiamo di utilizzare CAD

Reply:
Già in un punto sotto veniva usato CAD

Reply:
ok... come ti dicevo prima va bene CAD